### PR TITLE
[New Rule] SoftwareUpdate Preferences Modification

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/868

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/defense_evasion_apple_softupdates_modification.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+security updates.
+"""
+false_positives = ["Authorized SoftwareUpdate Settings Changes"]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "SoftwareUpdate Preferences Modification"
+references = ["https://blog.checkpoint.com/2017/07/13/osxdok-refuses-go-away-money/"]
+risk_score = 43
```
Comment (bm11100) [inline rules/macos/defense_evasion_apple_softupdates_modification.toml:19]:
```suggestion
risk_score = 47
```

File: rules/macos/defense_evasion_apple_softupdates_modification.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2021/01/15"
+maturity = "production"
+updated_date = "2021/01/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies changes to SoftwareUpdate preferences using the built-in defaults command. Adversaries may attempt disable
+security updates.
```
Comment (bm11100) [inline rules/macos/defense_evasion_apple_softupdates_modification.toml:10]:
```suggestion
Identifies changes to the SoftwareUpdate preferences using the built-in defaults command. Adversaries may abuse this in an attempt to disable security updates.
```

Comment (bm11100) [reply rules/macos/defense_evasion_apple_softupdates_modification.toml:10]: re-lint

**Review by bm11100 — APPROVED** [review]
couple comments, otherwise LGTM

File: rules/macos/defense_evasion_apple_softupdates_modification.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2021/01/15"
+maturity = "production"
+updated_date = "2021/01/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies changes to the SoftwareUpdate preferences using the built-in defaults command. Adversaries may abuse this in an attempt to disable security updates.
```
Comment (threat-punter) [inline rules/macos/defense_evasion_apple_softupdates_modification.toml:9]: This rule needs to be linted

**Review by threat-punter — APPROVED** [review]
👍

**Pull request merged**